test/e2e/components/vl-map.js

```diff
@@ -1,6 +1,70 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+const VlMapBaseLayer = require('./vl-map-baselayer');
+const VlMapSearch = require('./vl-map-search');
 
-class VlMap extends VlElement {  
+class VlMap extends VlElement {
+	async _getMap() {
+		return this.shadowRoot;
+	}
+
+	async getBaseLayers() {
+		const childElements = await this.findElements(By.css(this.selector + ' *'));
```

Comment:
this.selector bestaat niet indien er aan de VlElement constructor geen string maar een webelement werd meegegeven